src/js/helpers/contractGrantActivityHelper.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Created By Jonathan Hill 03/26/20
+ */
+
+import { cloneDeep } from 'lodash';
+import { isAwardFinancialAssistance } from 'helpers/awardSummaryHelper';
+import { badPotentialEndDate } from '../../../tests/testResources/mockContractGrantActivityHelper';
+
+/**
+ * lineHelper
+ * - determines if a line should be drawn
+ * @param {Moment{}} date
+ * @returns {null || Number}
+ */
+export const lineHelper = (date) => {
+    if (!date || isNaN(date.valueOf())) return null;
+    return date.valueOf();
+};
+/**
+ * isBadData
+ * - determines if we do not have suitable data to draw the chart based on dates,
+ * award type and transactions
+ * @param {Object} dates - award dates
+ * @param {string} awardType - award type
+ * @param {Object[]} transactions - array of transaction objects
+ * @returns {Boolean} - is the data suitable for charting
+ */
+export const isBadData = (dates, awardType, transactions) => {
```

Comment:
It's not very readable to see a function like this unless you're in the weeds of the code. It's nice to have these comments BTW which helps but having a precisely named `fn` helps even more for readability.

```suggestion
export const areTransactionDatesOrAwardAmountsInvalid = (dates, awardType, transactions) => {
```